packages/client/src/pages/MainPage/MainPage.tsx

```diff
@@ -47,7 +47,11 @@ const MainPage: FC = () => {
       </StFlex>
 
       {isOpen && (
-        <Modal title="Правила игры" handleCloseModal={handleCloseModal}>
+        <Modal
+          title="Правила игры"
+          handleCloseModal={handleCloseModal}
+          isCloseOutside={true}
+          isCrossButton={true}>
```

Comment:
```suggestion
          isCloseOutside
          isCrossButton>
```
когда значение props true, обычно его не указывают (ну или можно не указывать)